aries-site/src/data/highlights.js

```diff
@@ -0,0 +1,93 @@
+export const highlights = [
+  {
+    name: 'Color',
```

Comment:
Same as above comment, any way we could simplify this to an array of strings.

The new data items such as "summary"/"image" that are specific to a modified card layout could be named something that wouldn't conflict with the existing keys of the data objects (if conflicts arise with existing data structure).

Reply:
For this case I considered that. But the reason I didn't is it seems like this is something we'd tweak each release and it would be hard to track down and clean up all of these where they really get nothing from the card. They simply share a name so one links to the other. The featured case may make more sense to put that data in the card data, but again, they have nothing in common other than one links to the other. One other issue I ran into is the 'What's New' label really needs the embedded apostrophe or it just looks wrong.

Reply:
I think that's fair, especially from a clean up perspective. Happy to leave as is for now.